src/lib.rs

```diff
@@ -33,6 +33,19 @@ mod log_stubs;
 #[macro_use]
 mod extra_assertions;
 
+/// Print all the warning messages raised while generating the bindings in a build script.
+///
+/// If you are using `bindgen` outside of a build script you should use [`Bindings::take_warnings`]
+/// directly instead.
+#[macro_export]
+macro_rules! print_warnings {
```

Comment:
I also don't see why this needs to be a macro, can it just be a public method on `Bindings`?

Reply:
I don't have any way to test it. But apparently `crates.io` dependencies won't emit build script warnings unless `-vv` is passed to `cargo`. I don't know if functions from a dependency fall in this category and wrote a macro to fall into the safe side

Reply:
If we add a method for this we wouldn't actually need to have a `warnings` method.